site2/docs/concepts-architecture-overview.md

```diff
@@ -139,23 +139,3 @@ Some important things to know about the Pulsar proxy:
 * Connecting clients don't need to provide *any* specific configuration to use the Pulsar proxy. You won't need to update the client configuration for existing applications beyond updating the IP used for the service URL (for example if you're running a load balancer over the Pulsar proxy).
 * [TLS encryption](security-tls-transport.md) and [authentication](security-tls-authentication.md) is supported by the Pulsar proxy
 
-## Service discovery
```

Comment:
Same comment as `site2/docs/deploy-bare-metal-multi-cluster.md`, but in this section, we can remove `Pulsar provides a built-in service discovery mechanism that you can set up using the instructions in the [Deploying a Pulsar instance](deploy-bare-metal.md#service-discovery-setup) guide.`